app/src/main/java/com/teamagam/gimelgimel/app/notifications/AppNotifier.java

```diff
@@ -0,0 +1,93 @@
+package com.teamagam.gimelgimel.app.notifications;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.app.NotificationCompat;
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.utils.Constants;
+import com.teamagam.gimelgimel.app.common.utils.Environment;
+import com.teamagam.gimelgimel.app.mainActivity.view.MainActivity;
+import com.teamagam.gimelgimel.domain.messages.entity.ChatMessage;
+import com.teamagam.gimelgimel.domain.messages.entity.features.AlertFeature;
+import com.teamagam.gimelgimel.domain.notifications.NotifyOnNewMessageInteractor;
+
+import static android.support.v4.app.NotificationCompat.PRIORITY_MAX;
+
+public class AppNotifier implements NotifyOnNewMessageInteractor.NotificationDisplayer {
+
+  private static final String ALERT_TITLE = "New Alert!";
```

Comment:
All these should reside in the strings xml file (and have a matching translations)